__tests__/setup/createDirectLineOptions.js

```diff
@@ -0,0 +1,67 @@
+import fetch from 'node-fetch';
+
+const {
+  DEFAULT_USER_ID = 'u-12345',
```

Comment:
Note, if tests are using enhanced authentication, the username needs to start with `dl_`. Might be good to just make that change now.